Src/BootCamp.Chapter/BinaryConverter.cs

```diff
@@ -1,19 +1,73 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public static class BinaryConverter
     {
+        /// <summary>
+        /// Convert binary to Integer
+        /// </summary>
+        /// <param name="binary">binary string</param>
+        /// <returns>int number</returns>
         public static long ToInteger(string binary)
         {
-            return 0;
+            if (!IsValidBinaryNumber(binary))
+                return 0;
+
+            int intResult = 0;
+            var j = 0; 
+
+            for (int i = binary.Length - 1; i >= 0; i--, j++)
```

Comment:
Quite cool. Though I would avoid naming variable in a single letter.
`j` should be renamed to `rank`, `i` should be renamed to `indexOfDigit`